dynawo/sources/ModelicaCompiler/test/ModelicaModel/reference/GeneratorPQ_Dyn.cpp

```diff
@@ -577,6 +577,7 @@ void ModelGeneratorPQ_Dyn::setY0omc()
 
 void ModelGeneratorPQ_Dyn::callCustomParametersConstructors()
 {
+  // Delays
```

Comment:
Please remove this useless comment when there is no delay

Reply:
done